py-utils/src/utils/msg_bus/message_bus/template/factory.py

```diff
@@ -0,0 +1,25 @@
+from message_bus.adaptee import KafkaAdaptee, ConfluentAdaptee
+from message_bus.config import KafkaConfig, ConfluentKafkaConfig
+
+class Factory(object):
+
+    def __init__(self, adaptors):
+        self.m_dict = adaptors
+
+    def __call__(self, mq_name):
+        return self.m_dict[mq_name]()
+
+class kafkaFactory(Factory):
```

Comment:
- This class should be called "MessageBrokerFactory". 
- Template should be generic and not have anything specific to any modules. Move this part of the code to message_bus/message_broker.py along with MessageBroker class.